runners/katacoda/index.ts

```diff
@@ -205,7 +205,7 @@ export class Katacoda extends Runner {
         }
         this.pushStep(runCommand, "Create a new file", "step" + runCommand.stepIndex + ".md");
         
-        this.renderTemplate("createFile.md", this.outputPathTutorial + "step" + runCommand.stepIndex + ".md", { text: runCommand.text, textAfter: runCommand.textAfter, filePath: filePath, fileDir: fileDir , fileName:fileName , content: content});
```

Comment:
I am not sure, but did you delete the outer bracket?

Reply:
Yes